jerry-core/lit/lit-char-helpers.c

```diff
@@ -98,6 +101,43 @@ search_char_in_interval_array (ecma_char_t c,               /**< code unit */
 } /* search_char_in_interval_array */
 
 /**
+ * Print a specified character.
+ */
+void
+lit_char_put (ecma_char_t c) /**< code unit */
+{
+  JERRY_STATIC_ASSERT (sizeof (c) == 2,
+                       size_of_code_point_must_be_equal_to_2_bytes);
+
+  if (c != LIT_CHAR_NULL && c <= LIT_UTF8_1_BYTE_CODE_POINT_MAX)
+  {
+    jerry_port_putchar ((char) c);
+  }
+  else
+  {
+    uint32_t byte_high = (uint32_t) JRT_EXTRACT_BIT_FIELD (ecma_char_t, c,
+                                                           JERRY_BITSINBYTE,
+                                                           JERRY_BITSINBYTE);
+    uint32_t byte_low = (uint32_t) JRT_EXTRACT_BIT_FIELD (ecma_char_t, c,
+                                                          0,
+                                                          JERRY_BITSINBYTE);
+
+
+    jerry_port_putchar ('\\');
+    jerry_port_putchar ('u');
+    // print high byte
+    char out_buf[3];
+    itoa(byte_high, out_buf, 16);
```

Comment:
Looks like neither itoa nor sprintf is implemented in jerry-libc. What's the recommended solution here, I would happily add an itoa implementation.